source/shared/cpp/ObjectModel/HostConfig.cpp

```diff
@@ -631,16 +631,16 @@ unsigned int HostConfig::GetFontSize(FontType fontType, TextSize size) const
     // desired font size
     auto result = GetFontType(fontType).fontSizes.GetFontSize(size);
 
-    // UINT_MAX used to check if value was defined
-    if (result == UINT_MAX)
+    // std::numeric_limits<unsigned int>::max() used to check if value was defined
```

Comment:
Do we care about / should we care about the differences of `unsigned int` and `uint32_t` and the like?

Reply:
I don't think we really care in this *specific* case since we're using this as a sentinel value (and presumably we'll hit platform font size limits before we approach the upper limit of `unsigned int` regardless of how many bits it has). additionally, this is strictly for HostConfig, so it's not controllable outside of the host.